packages/logs/src/domain/trackConsoleLogs.ts

```diff
@@ -0,0 +1,49 @@
+import type { ClocksState } from '@datadog/browser-core'
+import { instrumentMethodAndCallOriginal, clocksNow, formatConsoleParameters } from '@datadog/browser-core'
+import type { StatusType } from './logger'
+
+export const ApiName = {
+  log: 'log',
+  debug: 'debug',
+  info: 'info',
+  warn: 'warn',
+} as const
+
+export const LogStatus = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏more precise naming
```suggestion
export const LogStatusForApi = {
```